gadget/install/install.go

```diff
@@ -569,7 +570,10 @@ func MountVolumes(onVolumes map[string]*gadget.Volume, encSetupData *EncryptionS
 		for _, mntPt := range mountPoints {
 			errUnmount := sysUnmount(mntPt, 0)
 			if errUnmount != nil {
-				logger.Noticef("cannot unmount %q: %v", mntPt, errUnmount)
+				logger.Noticef("cannot unmount %q: %v (trying lazy unmount next)", mntPt, errUnmount)
+				// lazy umount on error, see LP:2025402
+				errUnmount = sysUnmount(mntPt, syscall.MNT_DETACH)
+				logger.Noticef("cannot lazy unmount %q: %v", mntPt, errUnmount)
```

Comment:
nitpick: should we turn this into some sort of helper? as the code in content is also similar

Reply:
Great point, I will do a small followup